Code/Common/include/sitkImage.h

```diff
@@ -400,8 +400,11 @@ namespace simple
      *
      * The pointer to the buffer is not referenced
      * counted. Additionally, while this image is made unique before
-     * returnign the pointer, additional copying and usage may
-     * introduce unexpected aliasing.
+     * returning the pointer, additional copying and usage may
+     * introduce unexpected aliasing if the image's buffer.
```

Comment:
Sentence seems to end unexpectedly, probably should end "additional copying and usage may introduce unexpected aliasing."

Reply:
if->of